developer/src/kmc-ldml/src/compiler/vars.ts

```diff
@@ -130,9 +133,60 @@ export class VarsCompiler extends SectionCompiler {
       }));
       valid = false;
     }
+
+    valid = this.validateMarkers() && valid; // accumulate validity
+
     return valid;
   }
 
+  private collectMarkers(emitMarkers : Set<string>, matchMarkers : Set<string>) : boolean {
+    let valid = true;
+
+    // call our friends to validate
+    valid = this.validateVarsMarkers(this.keyboard, emitMarkers, matchMarkers) && valid; // accumulate validity
+    valid = KeysCompiler.validateMarkers(this.keyboard, emitMarkers, matchMarkers) && valid; // accumulate validity
+    valid = TransformCompiler.validateMarkers(this.keyboard, emitMarkers, matchMarkers) && valid; // accumulate validity
+    valid = DispCompiler.validateMarkers(this.keyboard, emitMarkers, matchMarkers) && valid; // accumulate validity
```

Comment:
so i'm trying to keep the locality of code here.  so `disp.ts` is still responsible for understanding the `<display>` element etc